packages/sdk/src/chainConfig/assets/polygon.ts

```diff
@@ -218,28 +218,28 @@ export const assets: SupportedAsset[] = [
   },
   {
     symbol: assetSymbols["JEUR-PAR"],
-    underlying: "0x0f110c55efe62c16d553a3d3464b77e1853d0e97",
+    underlying: "0x0f110c55EfE62c16D553A3d3464B77e1853d0e97",
```

Comment:
are these case changes intentional?

Reply:
Yes, usually the ui works with the addresses direclty from the contract and never lowercases them. So we should use the correctly formatted address here.